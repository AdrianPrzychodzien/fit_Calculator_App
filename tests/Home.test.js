import React from 'react'
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer'
import configureStore from 'redux-mock-store'
import Home from '../screens/Home'

const mockStore = configureStore([]);

describe('Home', () => {
  let store;

  beforeEach(() => {
    store = mockStore({
      data: {
        height: 180,
        weight: 80,
        age: 20,
        sex: "Male",
        lifeActivity: 4,
        formula: "MifflinStJeor"
      }
    });
  });

  test('snapshot renders', () => {
    const component = renderer.create(
      <Provider store={store}>
        <Home />
      </Provider>
    )
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})