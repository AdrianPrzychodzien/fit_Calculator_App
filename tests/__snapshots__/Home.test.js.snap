// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home snapshot renders 1`] = `
<RCTScrollView>
  <View>
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "flexDirection": "column",
          "justifyContent": "center",
          "padding": 30,
          "width": "100%",
        }
      }
    >
      <Text
        style={
          Object {
            "fontSize": 26,
            "fontWeight": "bold",
            "paddingVertical": 5,
            "textAlign": "center",
          }
        }
      >
        Home page
      </Text>
      <View>
        <Text
          style={
            Object {
              "fontSize": 18,
              "lineHeight": 28,
              "paddingVertical": 10,
              "textAlign": "center",
            }
          }
        >
          You are a 
          20
           year old 
          Male
           who is 
          180
           tall and weights
           
          80
           kg while 
          doing Heavy Exercise
        </Text>
      </View>
      <Text
        style={
          Object {
            "fontSize": 16,
            "marginTop": 15,
            "textAlign": "center",
          }
        }
      >
        Which formula should be used for calculations?
      </Text>
      <View
        style={
          Object {
            "paddingVertical": 10,
          }
        }
      >
        <View
          style={
            Array [
              undefined,
              undefined,
              Object {
                "flexDirection": "row",
              },
            ]
          }
        >
          <View>
            <View
              style={
                Array [
                  Object {
                    "flexDirection": "row",
                    "marginBottom": 5,
                  },
                  undefined,
                  Object {
                    "flexDirection": "column",
                    "paddingLeft": 10,
                  },
                ]
              }
            >
              <View
                style={
                  Object {
                    "marginLeft": 0,
                  }
                }
              >
                <View
                  accessible={true}
                  focusable={true}
                  isTVSelectable={true}
                  onClick={[Function]}
                  onResponderGrant={[Function]}
                  onResponderMove={[Function]}
                  onResponderRelease={[Function]}
                  onResponderTerminate={[Function]}
                  onResponderTerminationRequest={[Function]}
                  onStartShouldSetResponder={[Function]}
                  style={
                    Object {
                      "alignItems": "center",
                      "alignSelf": "center",
                      "borderColor": "#428bca",
                      "borderRadius": 10,
                      "borderWidth": 3,
                      "height": 20,
                      "justifyContent": "center",
                      "opacity": 1,
                      "width": 20,
                    }
                  }
                >
                  <View
                    style={
                      Array [
                        Object {
                          "borderRadius": 10,
                        },
                        false,
                        false,
                        false,
                      ]
                    }
                  />
                </View>
              </View>
              <View
                accessible={true}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Array [
                    Object {},
                    Object {
                      "alignItems": "center",
                      "alignSelf": "center",
                      "flexDirection": "row",
                    },
                  ]
                }
              >
                <Text
                  style={
                    Array [
                      Object {
                        "lineHeight": 20,
                        "paddingLeft": 10,
                      },
                      false,
                      Object {
                        "color": undefined,
                      },
                      Object {
                        "fontSize": 18,
                        "paddingVertical": 10,
                      },
                    ]
                  }
                >
                  MifflinStJeor
                </Text>
              </View>
            </View>
          </View>
          <View>
            <View
              style={
                Array [
                  Object {
                    "flexDirection": "row",
                    "marginBottom": 5,
                  },
                  undefined,
                  Object {
                    "flexDirection": "column",
                    "paddingLeft": 10,
                  },
                ]
              }
            >
              <View
                style={
                  Object {
                    "marginLeft": 0,
                  }
                }
              >
                <View
                  accessible={true}
                  focusable={true}
                  isTVSelectable={true}
                  onClick={[Function]}
                  onResponderGrant={[Function]}
                  onResponderMove={[Function]}
                  onResponderRelease={[Function]}
                  onResponderTerminate={[Function]}
                  onResponderTerminationRequest={[Function]}
                  onStartShouldSetResponder={[Function]}
                  style={
                    Object {
                      "alignItems": "center",
                      "alignSelf": "center",
                      "borderColor": "#428bca",
                      "borderRadius": 10,
                      "borderWidth": 3,
                      "height": 20,
                      "justifyContent": "center",
                      "opacity": 1,
                      "width": 20,
                    }
                  }
                >
                  <View
                    style={
                      Array [
                        Object {
                          "borderRadius": 10,
                        },
                        false,
                        false,
                        false,
                      ]
                    }
                  />
                </View>
              </View>
              <View
                accessible={true}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Array [
                    Object {},
                    Object {
                      "alignItems": "center",
                      "alignSelf": "center",
                      "flexDirection": "row",
                    },
                  ]
                }
              >
                <Text
                  style={
                    Array [
                      Object {
                        "lineHeight": 20,
                        "paddingLeft": 10,
                      },
                      false,
                      Object {
                        "color": undefined,
                      },
                      Object {
                        "fontSize": 18,
                        "paddingVertical": 10,
                      },
                    ]
                  }
                >
                  HarrisBenedict
                </Text>
              </View>
            </View>
          </View>
          <View>
            <View
              style={
                Array [
                  Object {
                    "flexDirection": "row",
                    "marginBottom": 5,
                  },
                  undefined,
                  Object {
                    "flexDirection": "column",
                    "paddingLeft": 10,
                  },
                ]
              }
            >
              <View
                style={
                  Object {
                    "marginLeft": 0,
                  }
                }
              >
                <View
                  accessible={true}
                  focusable={true}
                  isTVSelectable={true}
                  onClick={[Function]}
                  onResponderGrant={[Function]}
                  onResponderMove={[Function]}
                  onResponderRelease={[Function]}
                  onResponderTerminate={[Function]}
                  onResponderTerminationRequest={[Function]}
                  onStartShouldSetResponder={[Function]}
                  style={
                    Object {
                      "alignItems": "center",
                      "alignSelf": "center",
                      "borderColor": "#428bca",
                      "borderRadius": 10,
                      "borderWidth": 3,
                      "height": 20,
                      "justifyContent": "center",
                      "opacity": 1,
                      "width": 20,
                    }
                  }
                >
                  <View
                    style={
                      Array [
                        Object {
                          "borderRadius": 10,
                        },
                        false,
                        false,
                        false,
                      ]
                    }
                  />
                </View>
              </View>
              <View
                accessible={true}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Array [
                    Object {},
                    Object {
                      "alignItems": "center",
                      "alignSelf": "center",
                      "flexDirection": "row",
                    },
                  ]
                }
              >
                <Text
                  style={
                    Array [
                      Object {
                        "lineHeight": 20,
                        "paddingLeft": 10,
                      },
                      false,
                      Object {
                        "color": undefined,
                      },
                      Object {
                        "fontSize": 18,
                        "paddingVertical": 10,
                      },
                    ]
                  }
                >
                  KatchMcardle
                </Text>
              </View>
            </View>
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "paddingVertical": 10,
          }
        }
      >
        <View
          accessibilityRole="button"
          accessibilityStates={Array []}
          accessible={true}
          focusable={true}
          isTVSelectable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "opacity": 1,
            }
          }
        >
          <View
            style={
              Array [
                Object {},
              ]
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "color": "#007AFF",
                    "fontSize": 18,
                    "padding": 8,
                    "textAlign": "center",
                  },
                  Object {
                    "color": "#428bca",
                  },
                ]
              }
            >
              Calculate
            </Text>
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "paddingVertical": 15,
            "width": "90%",
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "flexDirection": "row",
              "justifyContent": "space-between",
              "paddingVertical": 15,
              "width": "100%",
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "flexDirection": "row",
              }
            }
          >
            <View
              style={
                Object {
                  "marginRight": 10,
                }
              }
            >
              <RNSVGSvgView
                align="xMidYMid"
                bbHeight={36}
                bbWidth={36}
                fill="#428bca"
                focusable={false}
                height={36}
                meetOrSlice={0}
                minX={0}
                minY={0}
                size={36}
                style={
                  Array [
                    Object {
                      "backgroundColor": "transparent",
                      "borderWidth": 0,
                    },
                    Object {},
                    Object {
                      "opacity": 1,
                    },
                    Object {
                      "flex": 0,
                      "height": 36,
                      "width": 36,
                    },
                  ]
                }
                vbHeight={512}
                vbWidth={640}
                width={36}
              >
                <RNSVGGroup
                  fill={
                    Array [
                      0,
                      4282551242,
                    ]
                  }
                  fillOpacity={1}
                  fillRule={1}
                  font={Object {}}
                  matrix={
                    Array [
                      1,
                      0,
                      0,
                      1,
                      0,
                      0,
                    ]
                  }
                  opacity={1}
                  originX={0}
                  originY={0}
                  propList={
                    Array [
                      "fill",
                    ]
                  }
                  rotation={0}
                  scaleX={1}
                  scaleY={1}
                  skewX={0}
                  skewY={0}
                  stroke={null}
                  strokeDasharray={null}
                  strokeDashoffset={null}
                  strokeLinecap={0}
                  strokeLinejoin={0}
                  strokeMiterlimit={4}
                  strokeOpacity={1}
                  strokeWidth={1}
                  vectorEffect={0}
                  x={0}
                  y={0}
                >
                  <RNSVGPath
                    d="M176 256c44.11 0 80-35.89 80-80s-35.89-80-80-80-80 35.89-80 80 35.89 80 80 80zm352-128H304c-8.84 0-16 7.16-16 16v144H64V80c0-8.84-7.16-16-16-16H16C7.16 64 0 71.16 0 80v352c0 8.84 7.16 16 16 16h32c8.84 0 16-7.16 16-16v-48h512v48c0 8.84 7.16 16 16 16h32c8.84 0 16-7.16 16-16V240c0-61.86-50.14-112-112-112z"
                    fill={
                      Array [
                        0,
                        4278190080,
                      ]
                    }
                    fillOpacity={1}
                    fillRule={1}
                    matrix={
                      Array [
                        1,
                        0,
                        0,
                        1,
                        0,
                        0,
                      ]
                    }
                    opacity={1}
                    originX={0}
                    originY={0}
                    propList={Array []}
                    rotation={0}
                    scaleX={1}
                    scaleY={1}
                    skewX={0}
                    skewY={0}
                    stroke={null}
                    strokeDasharray={null}
                    strokeDashoffset={null}
                    strokeLinecap={0}
                    strokeLinejoin={0}
                    strokeMiterlimit={4}
                    strokeOpacity={1}
                    strokeWidth={1}
                    vectorEffect={0}
                    x={0}
                    y={0}
                  />
                </RNSVGGroup>
              </RNSVGSvgView>
            </View>
            <Text
              style={
                Object {
                  "fontSize": 18,
                }
              }
            >
              Resting metabolic rate:
            </Text>
          </View>
          <Text
            style={
              Object {
                "fontSize": 18,
                "fontWeight": "bold",
              }
            }
          >
            1830
          </Text>
        </View>
        <View
          style={
            Object {
              "alignItems": "center",
              "flexDirection": "row",
              "justifyContent": "space-between",
              "paddingVertical": 15,
              "width": "100%",
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "flexDirection": "row",
              }
            }
          >
            <View
              style={
                Object {
                  "marginRight": 10,
                }
              }
            >
              <RNSVGSvgView
                align="xMidYMid"
                bbHeight={36}
                bbWidth={36}
                fill="#428bca"
                focusable={false}
                height={36}
                meetOrSlice={0}
                minX={0}
                minY={0}
                size={36}
                style={
                  Array [
                    Object {
                      "backgroundColor": "transparent",
                      "borderWidth": 0,
                    },
                    Object {},
                    Object {
                      "opacity": 1,
                    },
                    Object {
                      "flex": 0,
                      "height": 36,
                      "width": 36,
                    },
                  ]
                }
                vbHeight={512}
                vbWidth={416}
                width={36}
              >
                <RNSVGGroup
                  fill={
                    Array [
                      0,
                      4282551242,
                    ]
                  }
                  fillOpacity={1}
                  fillRule={1}
                  font={Object {}}
                  matrix={
                    Array [
                      1,
                      0,
                      0,
                      1,
                      0,
                      0,
                    ]
                  }
                  opacity={1}
                  originX={0}
                  originY={0}
                  propList={
                    Array [
                      "fill",
                    ]
                  }
                  rotation={0}
                  scaleX={1}
                  scaleY={1}
                  skewX={0}
                  skewY={0}
                  stroke={null}
                  strokeDasharray={null}
                  strokeDashoffset={null}
                  strokeLinecap={0}
                  strokeLinejoin={0}
                  strokeMiterlimit={4}
                  strokeOpacity={1}
                  strokeWidth={1}
                  vectorEffect={0}
                  x={0}
                  y={0}
                >
                  <RNSVGPath
                    d="M207.9 15.2c.8 4.7 16.1 94.5 16.1 128.8 0 52.3-27.8 89.6-68.9 104.6L168 486.7c.7 13.7-10.2 25.3-24 25.3H80c-13.7 0-24.7-11.5-24-25.3l12.9-238.1C27.7 233.6 0 196.2 0 144 0 109.6 15.3 19.9 16.1 15.2 19.3-5.1 61.4-5.4 64 16.3v141.2c1.3 3.4 15.1 3.2 16 0 1.4-25.3 7.9-139.2 8-141.8 3.3-20.8 44.7-20.8 47.9 0 .2 2.7 6.6 116.5 8 141.8.9 3.2 14.8 3.4 16 0V16.3c2.6-21.6 44.8-21.4 48-1.1zm119.2 285.7l-15 185.1c-1.2 14 9.9 26 23.9 26h56c13.3 0 24-10.7 24-24V24c0-13.2-10.7-24-24-24-82.5 0-221.4 178.5-64.9 300.9z"
                    fill={
                      Array [
                        0,
                        4278190080,
                      ]
                    }
                    fillOpacity={1}
                    fillRule={1}
                    matrix={
                      Array [
                        1,
                        0,
                        0,
                        1,
                        0,
                        0,
                      ]
                    }
                    opacity={1}
                    originX={0}
                    originY={0}
                    propList={Array []}
                    rotation={0}
                    scaleX={1}
                    scaleY={1}
                    skewX={0}
                    skewY={0}
                    stroke={null}
                    strokeDasharray={null}
                    strokeDashoffset={null}
                    strokeLinecap={0}
                    strokeLinejoin={0}
                    strokeMiterlimit={4}
                    strokeOpacity={1}
                    strokeWidth={1}
                    vectorEffect={0}
                    x={0}
                    y={0}
                  />
                </RNSVGGroup>
              </RNSVGSvgView>
            </View>
            <Text
              style={
                Object {
                  "fontSize": 18,
                }
              }
            >
              Caloric needs:
            </Text>
          </View>
          <Text
            style={
              Object {
                "fontSize": 18,
                "fontWeight": "bold",
              }
            }
          >
            3157
          </Text>
        </View>
        <View
          style={
            Object {
              "alignItems": "center",
              "flexDirection": "row",
              "justifyContent": "space-between",
              "paddingVertical": 15,
              "width": "100%",
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "flexDirection": "row",
              }
            }
          >
            <View
              style={
                Object {
                  "marginRight": 10,
                }
              }
            >
              <RNSVGSvgView
                align="xMidYMid"
                bbHeight={36}
                bbWidth={36}
                fill="#428bca"
                focusable={false}
                height={36}
                meetOrSlice={0}
                minX={0}
                minY={0}
                size={36}
                style={
                  Array [
                    Object {
                      "backgroundColor": "transparent",
                      "borderWidth": 0,
                    },
                    Object {},
                    Object {
                      "opacity": 1,
                    },
                    Object {
                      "flex": 0,
                      "height": 36,
                      "width": 36,
                    },
                  ]
                }
                vbHeight={512}
                vbWidth={640}
                width={36}
              >
                <RNSVGGroup
                  fill={
                    Array [
                      0,
                      4282551242,
                    ]
                  }
                  fillOpacity={1}
                  fillRule={1}
                  font={Object {}}
                  matrix={
                    Array [
                      1,
                      0,
                      0,
                      1,
                      0,
                      0,
                    ]
                  }
                  opacity={1}
                  originX={0}
                  originY={0}
                  propList={
                    Array [
                      "fill",
                    ]
                  }
                  rotation={0}
                  scaleX={1}
                  scaleY={1}
                  skewX={0}
                  skewY={0}
                  stroke={null}
                  strokeDasharray={null}
                  strokeDashoffset={null}
                  strokeLinecap={0}
                  strokeLinejoin={0}
                  strokeMiterlimit={4}
                  strokeOpacity={1}
                  strokeWidth={1}
                  vectorEffect={0}
                  x={0}
                  y={0}
                >
                  <RNSVGPath
                    d="M96 464v32c0 8.84 7.16 16 16 16h224c8.84 0 16-7.16 16-16V153.25c4.56-2 8.92-4.35 12.99-7.12l142.05 47.63c8.38 2.81 17.45-1.71 20.26-10.08l10.17-30.34c2.81-8.38-1.71-17.45-10.08-20.26l-128.4-43.05c.42-3.32 1.01-6.6 1.01-10.03 0-44.18-35.82-80-80-80-29.69 0-55.3 16.36-69.11 40.37L132.96.83c-8.38-2.81-17.45 1.71-20.26 10.08l-10.17 30.34c-2.81 8.38 1.71 17.45 10.08 20.26l132 44.26c7.28 21.25 22.96 38.54 43.38 47.47V448H112c-8.84 0-16 7.16-16 16zM0 304c0 44.18 57.31 80 128 80s128-35.82 128-80h-.02c0-15.67 2.08-7.25-85.05-181.51-17.68-35.36-68.22-35.29-85.87 0C-1.32 295.27.02 287.82.02 304H0zm56-16l72-144 72 144H56zm328.02 144H384c0 44.18 57.31 80 128 80s128-35.82 128-80h-.02c0-15.67 2.08-7.25-85.05-181.51-17.68-35.36-68.22-35.29-85.87 0-86.38 172.78-85.04 165.33-85.04 181.51zM440 416l72-144 72 144H440z"
                    fill={
                      Array [
                        0,
                        4278190080,
                      ]
                    }
                    fillOpacity={1}
                    fillRule={1}
                    matrix={
                      Array [
                        1,
                        0,
                        0,
                        1,
                        0,
                        0,
                      ]
                    }
                    opacity={1}
                    originX={0}
                    originY={0}
                    propList={Array []}
                    rotation={0}
                    scaleX={1}
                    scaleY={1}
                    skewX={0}
                    skewY={0}
                    stroke={null}
                    strokeDasharray={null}
                    strokeDashoffset={null}
                    strokeLinecap={0}
                    strokeLinejoin={0}
                    strokeMiterlimit={4}
                    strokeOpacity={1}
                    strokeWidth={1}
                    vectorEffect={0}
                    x={0}
                    y={0}
                  />
                </RNSVGGroup>
              </RNSVGSvgView>
            </View>
            <Text
              style={
                Object {
                  "fontSize": 18,
                }
              }
            >
              Body mass index:
            </Text>
          </View>
          <Text
            style={
              Object {
                "fontSize": 18,
                "fontWeight": "bold",
              }
            }
          >
            24.69
          </Text>
        </View>
        <View
          style={
            Object {
              "alignItems": "center",
              "flexDirection": "row",
              "justifyContent": "space-between",
              "paddingVertical": 15,
              "width": "100%",
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "flexDirection": "row",
              }
            }
          >
            <View
              style={
                Object {
                  "marginRight": 10,
                }
              }
            >
              <RNSVGSvgView
                align="xMidYMid"
                bbHeight={36}
                bbWidth={36}
                fill="#428bca"
                focusable={false}
                height={36}
                meetOrSlice={0}
                minX={0}
                minY={0}
                size={36}
                style={
                  Array [
                    Object {
                      "backgroundColor": "transparent",
                      "borderWidth": 0,
                    },
                    Object {},
                    Object {
                      "opacity": 1,
                    },
                    Object {
                      "flex": 0,
                      "height": 36,
                      "width": 36,
                    },
                  ]
                }
                vbHeight={512}
                vbWidth={512}
                width={36}
              >
                <RNSVGGroup
                  fill={
                    Array [
                      0,
                      4282551242,
                    ]
                  }
                  fillOpacity={1}
                  fillRule={1}
                  font={Object {}}
                  matrix={
                    Array [
                      1,
                      0,
                      0,
                      1,
                      0,
                      0,
                    ]
                  }
                  opacity={1}
                  originX={0}
                  originY={0}
                  propList={
                    Array [
                      "fill",
                    ]
                  }
                  rotation={0}
                  scaleX={1}
                  scaleY={1}
                  skewX={0}
                  skewY={0}
                  stroke={null}
                  strokeDasharray={null}
                  strokeDashoffset={null}
                  strokeLinecap={0}
                  strokeLinejoin={0}
                  strokeMiterlimit={4}
                  strokeOpacity={1}
                  strokeWidth={1}
                  vectorEffect={0}
                  x={0}
                  y={0}
                >
                  <RNSVGPath
                    d="M320.2 243.8l-49.7 99.4c-6 12.1-23.4 11.7-28.9-.6l-56.9-126.3-30 71.7H60.6l182.5 186.5c7.1 7.3 18.6 7.3 25.7 0L451.4 288H342.3l-22.1-44.2zM473.7 73.9l-2.4-2.5c-51.5-52.6-135.8-52.6-187.4 0L256 100l-27.9-28.5c-51.5-52.7-135.9-52.7-187.4 0l-2.4 2.4C-10.4 123.7-12.5 203 31 256h102.4l35.9-86.2c5.4-12.9 23.6-13.2 29.4-.4l58.2 129.3 49-97.9c5.9-11.8 22.7-11.8 28.6 0l27.6 55.2H481c43.5-53 41.4-132.3-7.3-182.1z"
                    fill={
                      Array [
                        0,
                        4278190080,
                      ]
                    }
                    fillOpacity={1}
                    fillRule={1}
                    matrix={
                      Array [
                        1,
                        0,
                        0,
                        1,
                        0,
                        0,
                      ]
                    }
                    opacity={1}
                    originX={0}
                    originY={0}
                    propList={Array []}
                    rotation={0}
                    scaleX={1}
                    scaleY={1}
                    skewX={0}
                    skewY={0}
                    stroke={null}
                    strokeDasharray={null}
                    strokeDashoffset={null}
                    strokeLinecap={0}
                    strokeLinejoin={0}
                    strokeMiterlimit={4}
                    strokeOpacity={1}
                    strokeWidth={1}
                    vectorEffect={0}
                    x={0}
                    y={0}
                  />
                </RNSVGGroup>
              </RNSVGSvgView>
            </View>
            <Text
              style={
                Object {
                  "fontSize": 18,
                }
              }
            >
              Maximum heart rates:
            </Text>
          </View>
          <Text
            style={
              Object {
                "fontSize": 18,
                "fontWeight": "bold",
              }
            }
          >
            192
          </Text>
        </View>
        <View
          style={
            Object {
              "alignItems": "center",
              "flexDirection": "row",
              "justifyContent": "space-between",
              "paddingVertical": 15,
              "width": "100%",
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "flexDirection": "row",
              }
            }
          >
            <View
              style={
                Object {
                  "marginRight": 10,
                }
              }
            >
              <RNSVGSvgView
                align="xMidYMid"
                bbHeight={36}
                bbWidth={36}
                fill="#428bca"
                focusable={false}
                height={36}
                meetOrSlice={0}
                minX={0}
                minY={0}
                size={36}
                style={
                  Array [
                    Object {
                      "backgroundColor": "transparent",
                      "borderWidth": 0,
                    },
                    Object {},
                    Object {
                      "opacity": 1,
                    },
                    Object {
                      "flex": 0,
                      "height": 36,
                      "width": 36,
                    },
                  ]
                }
                vbHeight={512}
                vbWidth={416}
                width={36}
              >
                <RNSVGGroup
                  fill={
                    Array [
                      0,
                      4282551242,
                    ]
                  }
                  fillOpacity={1}
                  fillRule={1}
                  font={Object {}}
                  matrix={
                    Array [
                      1,
                      0,
                      0,
                      1,
                      0,
                      0,
                    ]
                  }
                  opacity={1}
                  originX={0}
                  originY={0}
                  propList={
                    Array [
                      "fill",
                    ]
                  }
                  rotation={0}
                  scaleX={1}
                  scaleY={1}
                  skewX={0}
                  skewY={0}
                  stroke={null}
                  strokeDasharray={null}
                  strokeDashoffset={null}
                  strokeLinecap={0}
                  strokeLinejoin={0}
                  strokeMiterlimit={4}
                  strokeOpacity={1}
                  strokeWidth={1}
                  vectorEffect={0}
                  x={0}
                  y={0}
                >
                  <RNSVGPath
                    d="M272 96c26.51 0 48-21.49 48-48S298.51 0 272 0s-48 21.49-48 48 21.49 48 48 48zM113.69 317.47l-14.8 34.52H32c-17.67 0-32 14.33-32 32s14.33 32 32 32h77.45c19.25 0 36.58-11.44 44.11-29.09l8.79-20.52-10.67-6.3c-17.32-10.23-30.06-25.37-37.99-42.61zM384 223.99h-44.03l-26.06-53.25c-12.5-25.55-35.45-44.23-61.78-50.94l-71.08-21.14c-28.3-6.8-57.77-.55-80.84 17.14l-39.67 30.41c-14.03 10.75-16.69 30.83-5.92 44.86s30.84 16.66 44.86 5.92l39.69-30.41c7.67-5.89 17.44-8 25.27-6.14l14.7 4.37-37.46 87.39c-12.62 29.48-1.31 64.01 26.3 80.31l84.98 50.17-27.47 87.73c-5.28 16.86 4.11 34.81 20.97 40.09 3.19 1 6.41 1.48 9.58 1.48 13.61 0 26.23-8.77 30.52-22.45l31.64-101.06c5.91-20.77-2.89-43.08-21.64-54.39l-61.24-36.14 31.31-78.28 20.27 41.43c8 16.34 24.92 26.89 43.11 26.89H384c17.67 0 32-14.33 32-32s-14.33-31.99-32-31.99z"
                    fill={
                      Array [
                        0,
                        4278190080,
                      ]
                    }
                    fillOpacity={1}
                    fillRule={1}
                    matrix={
                      Array [
                        1,
                        0,
                        0,
                        1,
                        0,
                        0,
                      ]
                    }
                    opacity={1}
                    originX={0}
                    originY={0}
                    propList={Array []}
                    rotation={0}
                    scaleX={1}
                    scaleY={1}
                    skewX={0}
                    skewY={0}
                    stroke={null}
                    strokeDasharray={null}
                    strokeDashoffset={null}
                    strokeLinecap={0}
                    strokeLinejoin={0}
                    strokeMiterlimit={4}
                    strokeOpacity={1}
                    strokeWidth={1}
                    vectorEffect={0}
                    x={0}
                    y={0}
                  />
                </RNSVGGroup>
              </RNSVGSvgView>
            </View>
            <Text
              style={
                Object {
                  "fontSize": 18,
                }
              }
            >
              Training heart rates:
            </Text>
          </View>
          <Text
            style={
              Object {
                "fontSize": 18,
                "fontWeight": "bold",
              }
            }
          >
            125-163
          </Text>
        </View>
      </View>
    </View>
  </View>
</RCTScrollView>
`;
